lib/sla.js

```diff
@@ -65,16 +91,29 @@ function addChannelsToBridge(channel, bridge) {
  *   addChannelsToBridge().
  * @return {Q} Q promise object.
  */
-function createChannel(client, bridge) {
+function createChannel(client, inbound, bridge) {
   var defer = Q.defer();
-  var channel = client.Channel();
-  var originate = Q.denodeify(channel.originate.bind(channel));
-  originate({endpoint: 'SIP/phone', app: 'sla'})
+  var dialed = client.Channel();
+  var originate = Q.denodeify(dialed.originate.bind(dialed));
+  originate({endpoint: 'SIP/phone', app: 'sla', appArgs: 'dialed'})
     .catch(function (err) {
       defer.reject(err);
     });
-  channel.once('StasisStart', function(event, channel) {
-    defer.resolve(channel);
+  dialed.on('StasisStart', function(event, dialed) {
```

Comment:
You should use once here. This will never happen more than once. Currently your event handler is going to stay in memory for the entire run of the program, essentially causing a memory leak.